cw/tvla.py

```diff
@@ -911,15 +955,57 @@ def run_tvla(ctx: typer.Context):
                     else:
                         plt.close()
         else:
-            #
             c = np.ones(num_samples)
+            xaxs = range(sample_start, sample_start + num_samples)
             fig, axs = plt.subplots(3, sharex=True)
-            axs[0].plot(single_trace, "k")
+            axs[0].set_title("TVLA of " + cfg["project_file"])
```

Comment:
Maybe we should strip off `projects/` from the header.

Reply:
Makes sense, thanks!

Reply:
I think this should be a (hardcoded) parameter. Stripping off is ok as long as we don't have multiple folders.